### ISSUE TITLE
Up button (move up)

### ISSUE THREAD

**Issue by ghost** [DC23/wde#17]
Would be more intuitive to have an Up button close to the navigation buttons (Back / Forward)

**Comment by DC23** [comment]
I agree. It shouldn't be too hard to move. I will add it to the next release.
